Z80Bench.cs

```diff
@@ -6,9 +6,9 @@
 
 namespace z80bench
 {
-    class Z80Bench
+    public class Z80Bench
```

Comment:
Had to loosen up the default visibility of your main class/method to call it from the tests.